src/views/ProfileView.vue

```diff
@@ -8,6 +8,21 @@
     </div>
     <div class="wrap-container">
       <div class="personal-info contain">
+        <div
+          v-if="errors.length"
+          class="errors"
+        >
+          <h4 class="errors-heading">Please correct the following problem<span v-if="errors.length > 1">s</span> before saving:</h4>
+          <ul class="errors-list">
+            <li v-for="error in errors">{{ error }}</li>
+          </ul>
+        </div>
+        <div
+          v-if="msg && msg !== 'Set!'"
```

Comment:
Is it necessary to keep using the `msg` property concept here? If possible, it would be good to do away with it entirely.

It looks like this component is the only one that looks for the presence/absence of "Set!" to determine what to do -- it's just a strange little communication system used by `ProfileView` and `UserService`. Think you could refactor around this?

Reply:
I've changed this to use the promise returned by `UserService` to set a boolean prop and display a more descriptive message than the one built into `UserService`. This way we don't need `ProfileView` to even have an `msg` property.

Reply:
Looks good!